front/pages/api/w/[wId]/assistant/agent_configurations/index.ts

```diff
@@ -222,19 +222,6 @@ async function handler(
         });
       }
 
-      if (
```

Comment:
Why remove this?

Reply:
<img width="543" alt="Screenshot 2024-05-13 at 20 33 39" src="https://github.com/dust-tt/dust/assets/14199823/5da45baf-935b-4429-a4bb-91c5eb6d087e">


🤦